src/infrastructure/httpServer/HttpServer.ts

```diff
@@ -203,6 +190,40 @@ export class HttpServer extends ConnectorInfrastructure<HttpServerConfiguration>
         this.app.use(genericErrorHandler(this.connectorMode));
     }
 
+    private useApiKey() {
+        if (!this.configuration.apiKey) {
+            switch (this.connectorMode) {
+                case "debug":
+                    return;
+                case "production":
+                    throw new Error("No API key set in configuration. This is required in production mode.");
+            }
+        }
+
+        const apiKeyStrength = checkAPIKeyStrength(this.configuration.apiKey, [
+            { id: 0, value: "Very Weak", minDiversity: 0, minLength: 0 },
+            { id: 1, value: "Strong", minDiversity: 0, minLength: 30 }
+        ]);
```

Comment:
Do I get it right, that a password of 29 characters and a lot of digitis, special characters and uppercase and lowercase letters is now displayed as "Very Weak"? I get that it does not meet our requirements, but maybe we should call it a "Weak" password then if we don't want to have a finer subdivision of password strengths.

Reply:
I have to define a "low bound", the name here is irrelevant.